test/node/pipe.js

```diff
@@ -92,4 +100,48 @@ describe("request pipe", () => {
     });
     req.pipe(stream);
   });
+
+  it("should follow redirects", done => {
```

Comment:
added a test for pipes with a good redirect

Reply:
I realize now there is already a test for a good redirect in `./test/node/pipe-redirect.js`